### PR TITLE
docman route major update

### PR DESCRIPTION
**Pull request by magarnier**

### CODE REVIEW

File: languages/en/deployment-guide/intro.rst [hunk 1]
```diff
@@ -13,6 +13,16 @@ Tuleap 11.7
 
   Tuleap 11.7 is currently under development.
   
+Major update on Docman REST routes
+----------------------------------
+The route GET docman_items/id/docman_items was returning too much data:
```
Comment (robinsoc) [inline languages/en/deployment-guide/intro.rst]: docman_items/id/docman_items => docman_items/:id/docman_items

File: languages/en/deployment-guide/intro.rst [hunk 2]
```diff
@@ -13,6 +13,16 @@ Tuleap 11.7
...
+ - for link the link url were returned
+ - for embedded file, the embedded file content were returned
+ 
+Having such info directly accessible without querying docman_items/id prevented us from count user access so we did not have choice to break our API and remove this keys from the route results.
```
Comment (robinsoc) [inline languages/en/deployment-guide/intro.rst]: docman_items/id => docman_items/:id

from [...]  API => to take into account user accesses to a document. So , we had to change our API.

remove this keys => these keys

File: languages/en/deployment-guide/intro.rst [hunk 3]
```diff
@@ -13,6 +13,16 @@ Tuleap 11.7
...
+If you need to access them, please call docman_items/id.
```
Comment (robinsoc) [inline languages/en/deployment-guide/intro.rst]: If you need to access them => If you need to log access history of a document
docman_items/id =>  docman_items/:id.

File: languages/en/deployment-guide/intro.rst [hunk 4]
```diff
@@ -13,6 +13,16 @@ Tuleap 11.7
 
   Tuleap 11.7 is currently under development.
   
+Major update on Docman REST routes
```
Comment (robinsoc) [inline languages/en/deployment-guide/intro.rst]: Why the french documentation is not also updated ?

Comment (magarnier) [reply languages/en/deployment-guide/intro.rst]: French document does not contains deployement guide, only user guide

File: languages/en/deployment-guide/intro.rst (hunk 3 above)
Comment (magarnier) [reply languages/en/deployment-guide/intro.rst]: I disagree, this sentence is correct, what is important is the fact that you need to access to document content (link_url or embedded file content), the fact that route logs access is something technical an internal to Tuleap behaviour.

Nobody will ever call this route only to log a document access.

**Pull request merged**